scripts/regression-tests.sh

```diff
@@ -4,18 +4,6 @@ if [[ "$CI" != "true" ]]
 then
   # When running this script locally, compare the current branch to master
   COMMIT_RANGE="..master"
-
-elif [[ "$TRAVIS_PULL_REQUEST" != "false" ]]
-then
-  # If we are on a PR build, we can use TRAVIS_COMMIT_RANGE
-  COMMIT_RANGE=$TRAVIS_COMMIT_RANGE
-
-else
-  # If we are on a branch build, and it has been force pushed, then TRAVIS_PULL_REQUEST will
-  # not contain useful information for the branch build.
-  COMMIT_RANGE="origin/master...$TRAVIS_BRANCH"
-  git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-  git fetch origin master
 fi
 
 AVACMD="npm run regression -- -t"
```

Comment:
When ava was running tests in parallel process in travis, the tests became pretty flaky, so we had to locked down concurrency to 1 with `-c 1` (by default ava uses the number of "cores"). I removed that command line argument when we switched to parallelizing across VMs. I'm not sure how flaky the tests will be in this new Github action environment? Have you tested triggering the tests a few times in a row?

Reply:
I didn't notice the flakey test in the other approach https://github.com/w3c/aria-practices/compare/master...nschonni:ava-github-action, although I don't think the library Ava uses for parallelization supports GitHub Actions right now https://github.com/avajs/ava#parallel-runs-in-ci so it might not be triggering.
Could add those environmental variables back though if you think it is better

Reply:
Let's see how it goes with this in, if I see flakey tests we know a quick fix (to locked down the concurrency). It could be the throttled travis environment that lead to the flakey tests.